RealmModelGenerator/AttributeDetailViewController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  AttributeDetailViewController.swift
+//  RealmModelGenerator
+//
+//  Created by Zhaolong Zhong on 3/29/16.
+//  Copyright © 2016 QuarkWorks. All rights reserved.
+//
+
+import Cocoa
+
+@objc protocol AttributeDetailViewControllerDelegate {
+    optional func attributeDetailDidChange(attributeDetailViewController:AttributeDetailViewController)
+}
+
+class AttributeDetailViewController: NSViewController, AttributeDetailViewDelegate {
+    static let TAG = NSStringFromClass(AttributeDetailViewController)
+
+    @IBOutlet weak var attributeDetailView: AttributeDetailView!
+    weak var delegate: AttributeDetailViewControllerDelegate?
+    
+    var attribute: Attribute?
+    var defaultAttribute: Attribute? {
+        willSet(defaultAttribute) {
+            attribute = defaultAttribute
+        }
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        attributeDetailView.delegate = self
+    }
+    
+    override func viewWillAppear() {
+        if attribute != nil {
+            attributeDetailView.name = attribute!.name
+        }
+    }
+    
+    func attributeDetailView(attributeDetailView: AttributeDetailView, shouldChangeAttributeName name: String) -> Bool {
+        do {
+            try self.attribute!.setName(name)
+            self.delegate?.attributeDetailDidChange?(self)
+        } catch {
+            let alert = NSAlert()
```

Comment:
this doesn't need to be solved now but if this could be executed in one line that would be great